database/storage/injectbase.go

```diff
@@ -22,22 +23,27 @@ var _ Interface = &InjectBase{}
 
 // Get returns a database record.
 func (i *InjectBase) Get(key string) (record.Record, error) {
-	return nil, errNotImplemented
+	return nil, ErrNotImplemented
+}
+
+// Get returns a database record.
+func (i *InjectBase) GetMeta(key string) (*record.Meta, error) {
+	return nil, ErrNotImplemented
 }
```

Comment:
As this method is actually a requirement (because it will always be called) we should remove it from `InjectBase` to make sure all DB implementations must have this one defined!